### ISSUE TITLE
Unknown Error:

### ISSUE THREAD

**Issue by sawyerknox** [GCalToolkit/Sheets2GCal#37]
I am getting this on my google sheet, however it works on our 2023 version and there are no changes....
![Screen Shot 2024-01-11 at 3 16 14 PM](https://github.com/GCalToolkit/Sheets2GCal/assets/156379059/c05a1f89-9a82-45dd-a434-580e59a574be)

**Comment by GCalToolkit** [comment]
This will be easier to sort out by email - could you please get in touch at info at sheets2gcal dot com with some more information about what you were doing: for example, were you only creating events (and were they created) or also updating other events?

If you could send screenshots showing the sheet and specifically the rows for the events in the errors that would help too. If necessary I might need to see a copy of the sheet, or at least a similar sheet with no sensitive data that shows the same type of error. Thanks

**Comment by soniakhuga** [comment]
I am having the same issue

**Comment by GCalToolkit** [comment]
Are you using several Google accounts in the same browser?

If so please see
https://github.com/GCalToolkit/Sheets2GCal/issues/27#issuecomment-1677911842 for several possible causes and solutions.

If not, please get in touch directly as I suggested to the OP above.

**Comment by soniakhuga** [comment]
No, and the #27 solutions didn't help, sent the email & attached the spreadsheet, thanks!

**Comment by GCalToolkit** [comment]
Are you sure you sent these to the correct address? I have received anything...

**Comment by soniakhuga** [comment]
Is this the right address: info @ sheets2gcal . com ?

**Comment by GCalToolkit** [comment]
That's correct, but I've been through all my mail, spam etc. and there's nothing :-/

**Comment by jstampleman** [comment]
I can't get it work either.  I get this "Unknown Error" also.  I got an email confirming that access was granted to Sheets2GCal, but I see no more helpful error messages than that.

**Comment by GCalToolkit** [comment]
Hi, did you already check what I suggested above about logging out of other Google accounts, or trying in an incognito window?

If so, pleased contact me by email as I'll need to see screenshots of your Sheet, especially the Header (row 1) which might contain private information.

You can use the address on my profile or Sheets2GCal at GCalTools dot com.
